src/EPR.Calculator.Frontend/appsettings.Development.json

```diff
@@ -41,6 +41,8 @@
     "CalculationRunSettings": {
         "RunParameterYear": "2024-25",
         "CalculationRunNameApi": "http://localhost:5055/v1/CheckCalcNameExists",
-        "CalculationRunApi": "http://localhost:5055/v1/calculatorRun"
+        "CalculationRunApi": "http://localhost:5055/v1/calculatorRun",
+        "DownloadResultApi": "http://localhost:5055/v1/DownloadResult",
+        "DownloadResultTimeout":  30000
```

Comment:
Sorry bieng pedantic, can you change the parameterName to DownloadResultTimeoutInSeconds